client/src/pages/profile/index.tsx

```diff
@@ -87,16 +71,16 @@ export class ProfilePage extends React.Component<Props, State> {
           })),
       })) as CoreJsInterviewsData[];
 
-  private fetchData = async () => {
-    this.setState({ isLoading: true });
-
-    const { router, session } = this.props;
+  const checkIsProfileOwner = (githubId: string, requestedGithubId: string): boolean => {
+    return githubId === requestedGithubId;
+  };
 
+  const fetchData = async () => {
     try {
       const githubId = router.query ? (router.query.githubId as string) : undefined;
       const [profile, connections, { data }] = await Promise.all([
-        this.userService.getProfileInfo(githubId?.toLowerCase()),
-        this.notificationsService.getUserConnections().catch(() => []),
+        userService.getProfileInfo(githubId?.toLowerCase()),
+        notificationsService.getUserConnections().catch(() => []),
```

Comment:
array seems not the right thing to return here, connections is an object, well in js everything is an object), but maybe it should be actual object `({})` or `({} as object)`